src/GitHub.App/Services/PullRequestService.cs

```diff
@@ -172,6 +172,17 @@ public IObservable<BranchTrackingDetails> CalculateHistoryDivergence(ILocalRepos
             });
         }
 
+        public IObservable<TreeChanges> GetTreeChanges(ILocalRepositoryModel repository, IPullRequestModel pullRequest)
+        {
+            return Observable.Defer(async () =>
+            {
+                var repo = gitService.GetRepository(repository.LocalPath);
+                await gitClient.Fetch(repo, "origin");
+                var changes = await gitClient.Compare(repo, pullRequest.Base.Sha, pullRequest.Head.Sha, detectRenames: true);
```

Comment:
After thinking about this for the weekend, I think what I'm doing here is the best thing to do as firstly it reflects the underlying API. Secondly (and less importantly) should we need any other options adding later (there are a number of potential options), by using named parameters the call sites won't need to change.